### PR TITLE
clarify where index.js comes from

### PR DESCRIPTION
**Pull request by trosel**
re-order some instructions up front to off-load installation and project creation to installation page

### CODE REVIEW

**Comment by pygy** [comment]
Thanks for the PR, I'll review the whole thing this evening CET.

File: docs/simple-application.md [hunk 1]
```diff
@@ -4,7 +4,15 @@ Let's develop a simple application that covers some of the major aspects of Sing
...
 
-First let's create an entry point for the application. Create a file `index.html`:
+---
+
+We could create the entire application in a single Javascript file, but doing so would make it difficult to navigate the codebase later on. Instead, let's split the code into *modules*, and assemble these modules into a *bundle* `bin/app.js`.
+
+There are many ways to setup a bundler tool, but most are distributed via NPM. In fact, most modern Javascript libraries and tools are distributed that way, including Mithril. NPM stands for Node.js Package Manager. To download NPM, [install Node.js](https://nodejs.org/en/); NPM is installed automatically with it. Once you have Node.js and NPM installed, install Mithril and create the project skeleton.
+
+To install Mithril, follow the instructions in the [installation](installation.md) page. Once you have completed the seven steps outlined in [Quick Start with Webpack](https://mithril.js.org/installation.html#quick-start-with-webpack), we are ready to build upon the project skeleton that you have created.
```
Comment (pygy) [inline docs/simple-application.md]: Could you change the link to `[[Quick Start with Parcel]](...#quick-start-with-parcel)`?

Comment (WildYorkies) [reply docs/simple-application.md]: @pygy Is there such a link? I'm not seeing that anywhere. I just see the webpack heading and instructions.

Comment (pygy) [reply docs/simple-application.md]: @WildYorkies it is in the `next` branch, it hasn't been published yet

**Comment by pygy** [comment]
Thanks @WildYorkies. The docs will need a more thorough review, because the `simple-application.md` page actually expects a WebPack setup...

If you have the time to give it a crack, the help is appreciated, and otherwise I'll do it.

**Comment by isiahmeadows** [comment]
@pygy What's the status on this?

File: docs/simple-application.md [hunk 2]
```diff
@@ -4,7 +4,15 @@ Let's develop a simple application that covers some of the major aspects of Sing
...
 
-First let's create an entry point for the application. Create a file `index.html`:
+---
+
+We could create the entire application in a single Javascript file, but doing so would make it difficult to navigate the codebase later on. Instead, let's split the code into *modules*, and assemble these modules into a *bundle* `bin/app.js`.
+
+There are many ways to setup a bundler tool, but most are distributed via NPM. In fact, most modern Javascript libraries and tools are distributed that way, including Mithril. NPM stands for Node.js Package Manager. To download NPM, [install Node.js](https://nodejs.org/en/); NPM is installed automatically with it. Once you have Node.js and NPM installed, install Mithril and create the project skeleton.
+
+To install Mithril, follow the instructions in the [installation](installation.md) page. Once you have completed the seven steps outlined in [Quick Start with Parcel](https://mithril.js.org/installation.html#quick-start-with-parcel), we are ready to build upon the project skeleton that you have created.
```
Comment (StephanHoyer) [inline docs/simple-application.md]: Use relative paths
```suggestion
To install Mithril, follow the instructions in the [installation](installation.md) page. Once you have completed the seven steps outlined in [Quick Start with Parcel](installation.md#quick-start-with-parcel), we are ready to build upon the project skeleton that you have created.
```

**Pull request closed without merging**